cms/static/sass/bootstrap/_components.scss

```diff
@@ -0,0 +1,195 @@
+// Open edX: components
+// ====================
```

Comment:
General question here. It seems like you took the footer, header, etc files and copied them all in here - do you think it would be worth breaking them out into _footer-bootstrap.scss, _navigation-bootstrap.scss, etc. to more closely match the other implementation? If not, perhaps we should add a follow up story to rework the studio header and footer and toss it in the backlog.

I'm a bit worried that your comment on the _footer.scss file that "these styles replicate the studio styles directly rather than benefiting..." will remain there and we will forget to ever deprecate those files and convert it all to bootstrap.